config/webpack/build-aot.webpack.config.js

```diff
@@ -19,6 +19,10 @@ function getWebpackConfig(skyPagesConfig, argv) {
   let commonConfig = common.getWebpackConfig(skyPagesConfig, argv);
   commonConfig.entry = null;
 
+  // Enable source map generation
+  let devtool = 'hidden-source-map';
+  let sourceMap = true;
```

Comment:
Instead of creating variables, we're probably good to place these values into the config directly.
```
devtool: 'hidden-source-map'
```